book_variables.yml

```diff
@@ -0,0 +1,16 @@
+## INSTRUCTIONS
+# Insert variables to be loaded into Gitbook
+# They can be referenced via {{ book.importedVars.<variable_name> }} the following example
+#
+# ```markdown
+# This is the value of my variable {{ book.importedVars.<variable_name> }}
+#
+#```
+#
+
+mojaloop:
```

Comment:
I think you can remove these variables as they are not used in this repo.

Keep the file in case any vars need to be added though.

Reply:
Have updated as requested.